store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -2320,4 +2325,93 @@ public void testValidateEscapeCharWithProperValue() throws IOException {
       FileUtils.deleteDirectory(new File(path));
     }
   }
+
+  @Test
+  public void testWriteWithDifferentDataType() {
```

Comment:
No binary datatype here? how it is related to this PR?

Reply:
Added binary.